test/ProkkaAnnotation_metagenome_test.py

```diff
@@ -0,0 +1,190 @@
+        gff_path = self.scratch + "/metagenome.gff"
+        shutil.copyfile(gff, gff_path)
+        shutil.copyfile(fasta, fasta_path)
+        metagenome_ref = self.gfu.fasta_gff_to_metagenome({
+            "gff_file": {'path': gff_path},
+            "fasta_file": {'path': fasta_path},
+            "genome_name": output_name,
+            "workspace_name": self.getWsName(),
+            "generate_missing_genes": True
+        })['genome_ref']
+        return metagenome_ref
+
+    def create_assembly(self, fasta, output_name):
+        """copies to scratch then saves as assembly"""
+        fasta_path = self.scratch + "/temp_assembly.fa"
+        shutil.copyfile(fasta, fasta_path)
+        assembly_ref = self.au.save_assembly_from_fasta({
+            "file": {"path": fasta_path},
+            "workspace_name": self.getWsName(),
+            "assembly_name": output_name,
+            "type": "metagenome"
+        })
+        return assembly_ref
+
+    def _filter_gff(self, gff):
+        fasta = []
+        save = []
+        with open(gff) as f:
+            for l in f:
+                if '##FASTA' in l:
+                    for line in f:
+                        fasta.append(line)
+                    break
+                if '##' in l:
+                    continue
+                save.append(l)
+        gff_path = gff + "_edited.gff"
+        with open(gff_path, 'w') as f:
+            for l in save:
+                f.write(l.strip() + '\n')
+        fasta_path = gff  + "edited.fa"
+        with open(fasta_path, 'w') as f:
+            for l in fasta:
+                f.write(l.strip() + '\n')
+        return gff_path, fasta_path
+
+    def verify_output(self, ret):
+        self.assertTrue('output_metagenome_ref' in ret)
+        self.assertTrue('report_name' in ret)
+        self.assertTrue('report_ref' in ret)
+
+    @unittest.skip('x')
+    def test_annotate_metagenome_from_ama_ref(self):
+        ret = self.serviceImpl.annotate_metagenome(self.getContext(), {
+            "object_ref": "",
+            "output_workspace": self.getWsName(),
+            "output_metagenome_name": "test_metagenome_from_metagenome"
+        })[0]
+        self.verify_output(ret)
+
+    @unittest.skip('x')
+    def test_annotate_metagenome_from_annotated_metagenome_assembly_short(self):
+        metagenome_gff = "data/short_one.gff"
+        metagenome_fasta = "data/short_one.fa"
+        ref = self.create_metagenome(metagenome_gff, metagenome_fasta, 'metagenome_metagenome_short')
+        ret = self.serviceImpl.annotate_metagenome(self.getContext(), {
+            "object_ref": ref,
+            "output_workspace": self.getWsName(),
+            "output_metagenome_name": "test_metagenome_from_metagenome_short"
+        })[0]
+        self.verify_output(ret)
+
+    def test_annotate_metagenome_from_annotated_metagenome_assembly(self):
+        metagenome_gff = "data/59111.assembled.gff"
+        metagenome_fasta = "data/59111.assembled.fna"
```

Comment:
I can't run the tests without this file 59111.assembled.fna Can you please include it?